scheduler/scheduler.go

```diff
@@ -198,28 +196,36 @@ func (driver *MesosSchedulerDriver) init() error {
 	driver.messenger.Install(driver.slaveLost, &mesos.LostSlaveMessage{})
 	driver.messenger.Install(driver.frameworkMessageRcvd, &mesos.ExecutorToFrameworkMessage{})
 	driver.messenger.Install(driver.frameworkErrorRcvd, &mesos.FrameworkErrorMessage{})
-
-	driver.masterDetector.Detect(detector.AsMasterChanged(driver.masterDetected))
+	driver.messenger.Install(driver.handleMasterChanged, &mesos.InternalMasterChangeDetected{})
```

Comment:
I was wondering where you were going with these internal messages.  Makes sense now seeing these handlers.